awslambdaric/bootstrap.py

```diff
@@ -347,7 +379,7 @@ def log(self, msg):
 
     def log_error(self, message_lines):
         error_message = "\n".join(message_lines)
-        self.log(error_message)
+        self.log(error_message, log_level=logging.FATAL)
```

Comment:
I think this should be ERROR - at least this is what we use today:  `[ERROR] ZeroDivisionError: division by zero`